drivers/kurimudb-driver-nodejson/index.ts

```diff
@@ -0,0 +1,84 @@
+import { readFileSync, existsSync, writeFileSync } from 'fs';
+
+const path = "./kurimudb.json"; // TODO: Maybe read it from model?
+let data = {};
+let loaded = false;
+
+function load(): void {
+  if (loaded) {
+    return;
+  }
+  loaded = true;
+  if (existsSync(path)) { // TODO: Find a way to use async methods.

```

Comment:
Here is a solution:
```typescript
export async function exists(path: fs.PathLike): Promise<boolean> {
  try {
    await fs.promises.stat(path);
    return true;
  } catch {
    return false;
  }
}

```

Reply:
Yes, I know we can use try-catch and stat to replace existSync. Actually I'm not sure if there would be a place for async initialize. Currently I can only init(aka load the json) in driver's constructor, which cannot be async.